protocol/messenger.go

```diff
@@ -1337,6 +1339,31 @@ func (m *Messenger) watchExpiredMessages() {
 	}()
 }
 
+// watchUnmutedChats regularly checks for chats that should be unmuted
+func (m *Messenger) watchUnmutedChats() {
+	m.logger.Debug("watching unmuted chats")
+	go func() {
+		for {
+			select {
+			case <-time.After(3 * time.Second): // Poll every 3 seconds
+				response := &MessengerResponse{}
+				m.allChats.Range(func(chatID string, c *Chat) bool {
+					if time.Now().Unix() > c.MuteTill && c.MuteTill != 0 && c.Muted {
+						_ = m.persistence.UnmuteChat(c.ID)
```

Comment:
You need to handle the error here, I would stop the loop and return

Reply:
I returned `false`